libfastpm/neutrinos_lra.c

```diff
@@ -138,11 +140,11 @@ static void delta_tot_first_init(_delta_tot_table * const d_tot, const int nk_in
     init_neutrinos_lra(d_tot, nk_in, 1.0);
     /*Set the prefactor for delta_nu, and the units system, which is Mpc/s.*/
     d_tot->light = LIGHT;
-    d_tot->delta_nu_prefac = 1.5 * cosmo->Omega_m * HUBBLE * HUBBLE /d_tot->light;
+    d_tot->delta_nu_prefac = 1.5 * cosmo->Omega_m * HUBBLE * HUBBLE / d_tot->light; // adrian light not in paper?
```

Comment:
The paper uses c = 1 units!

Reply:
Right -- I was just making a note to check if all the factors of c are correct. Here there's a c, there's one in fselngth (which make sense), and another in deriv_prefac. I think it's all fine, but feel free to check.